packages/scaffold-utils/src/SolanaHelpersUtils.ts

```diff
@@ -0,0 +1,71 @@
+import { PresetsUtil } from './PresetsUtil.js'
+
+import type { CaipNetwork } from '@web3modal/core'
+import type { Chain, Provider } from './SolanaTypesUtil.js'
+
+export const SolHelpersUtil = {
+  getChain(chains: Chain[], chainId: string | null) {
+    const chain = chains.find(lChain => lChain.chainId === chainId)
```

Comment:
Let's separate the preperation-execution-conclusion parts of the function body with a new line. This will help us to keep them in a good pattern to make them more readable.

```suggestion
    const chain = chains.find(lChain => lChain.chainId === chainId)
    
```